src/components/views/rooms/RoomList.js

```diff
@@ -84,8 +86,8 @@ module.exports = React.createClass({
         cli.on("Group.myMembership", this._onGroupMyMembership);
 
         this._groupStores = {};
+        this._selectedTagsRoomIdsForGroup = {};
```

Comment:
comment please. what is this? what are the types of the keys and values? don't make me reverse-engineer the class.